.github/workflows/pytest.yml

```diff
@@ -28,7 +28,7 @@ jobs:
           cache-dependency-glob: "pyproject.toml"
       - name: install-reqs
         # Python3.8 is technically at end-of-life, so we don't test everything
-        run: uv pip install -e ".[tests, core]" backports.zoneinfo --system
+        run: uv pip install -e ".[pandas,polars,pyarrow]" backports.zoneinfo --group tests --system
```

Comment:
Only installing what is tested (not all `core`) (`constructors=pandas,pyarrow,polars[eager],polars[lazy]`)